webapp/app/forms/session_data.py

```diff
@@ -45,4 +52,9 @@ def deserialize_session_data(serialized_session, ttl: Optional[int] = None):
 
 
 def override_session_data(stored_data, session_data_identifier='form_data'):
-    session[session_data_identifier] = serialize_session_data(stored_data)
+    key = create_key_identifier_with_user_id(session_data_identifier)
+    FormDataController().save_to_redis(key, serialize_session_data(stored_data))
```

Comment:
In my head, I was thinking of sth like that to keep the functionality of having different types of session_data in one session:
```suggestion
    key = create_key_identifier_with_user_id(session_data_identifier)
    old_session_data = deserialize(FormDataController().get_from_redis(key))
    old_session_data[session_data_identifier] = stored_data
    FormDataController().save_to_redis(key, serialize_session_data(old_session_data))
```

However, I think I like the approach of creating the identifier in the way you do it even more as we use the redis directly 👍